ts/Core/Axis/Color/ColorAxis.ts

```diff
@@ -988,9 +1000,9 @@ namespace ColorAxis {
         options: object;
         drawLegendSymbol: typeof LegendSymbol['drawRectangle'];
         visible: boolean;
-        setState: Function;
+        setState: Point['setState'];
         isDataClass: true;
-        setVisible: () => void;
+        setVisible: Function;
```

Comment:
You can write `setVisible()` if you like to define it as an object-related function.